### DISCUSSION TITLE
Recommendation - Software to split videos without loss of quality - lossless-cut

### DISCUSSION THREAD

**Discussion by dellagustin-sap** [InnerSourceCommons/InnerSourceMarketing#262] [Show and tell]
Hi all, as moderator at the InnerSource Summit 2023 I have recorded the session locally as a backup for the organization team.
The recording from Zoom was one big `.mp4` file.
I wanted to split it before uploading them for processing by the org team, but without having to "re-econde" the video, for a faster process time but also to avoid quality loss.
After a quick research, I tried two options (both on Windows):
* Mp4Splitter - https://www.mp4joiner.org/en/
* lossless-cut - https://github.com/mifi/lossless-cut

Mp4Splitter worked to a certain extent, but the performance was not great and it was crashing if I tried to play the video in it.

https://github.com/mifi/lossless-cut worked very well, so I used it at the end. Its processing is very fast (generated all the new files in a couple of minutes) and the result was very good. It also had a good perfomance on the UI and it was very easy to use.

**Note:** You can BUY it from known stores, or download it for FREE directly from the links in the Download section at the `README.md`, just below the store buttons. Buying of course is a way to support the project.

**Comment by spier** [comment]
Thanks for documenting this @dellagustin-sap!
